### PR TITLE
Vault USD transfer

### PR DESCRIPTION
**Pull request by cgaspart**
Use the SDK to deposit USD in a vault.
Issue: #55 #39

### CODE REVIEW

File: examples/basic_vault_transfer.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from hyperliquid.utils import constants
+import example_utils
+
+
+def main():
+    address, info, exchange = example_utils.setup(constants.TESTNET_API_URL, skip_ws=True)
+
+    if exchange.account_address != exchange.wallet.address:
```
Comment (traderben) [inline examples/basic_vault_transfer.py:8]: remove this, agents are allowed to perform vault transfers

File: examples/basic_vault_transfer.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        raise Exception("Agents do not have permission to perform internal transfers")
+
+    # Transfer 5 usd to the HLP Vault for demonstration purposes
+    transfer_result = exchange.vault_usd_transfer("0xa15099a30bbf2e68942d6f4c43d70d04faeab0a0", True, 5000000)
```
Comment (traderben) [inline examples/basic_vault_transfer.py:12]: nit
```suggestion
    transfer_result = exchange.vault_usd_transfer("0xa15099a30bbf2e68942d6f4c43d70d04faeab0a0", True, 5_000_000)
```

Comment (traderben) [reply examples/basic_vault_transfer.py:12]: Also, can you select the hlp address based on if exchange is mainnet vs. testnet (or add a comment to change it if using a different environment)

**Review by traderben — CHANGES_REQUESTED** [review]
Thanks for creating this PR. Overall looks good, just a couple of small comments

**Comment by cgaspart** [comment]
@traderben I edited the exemple as requested.
Also I created a new variable that refer to the HLP vault so that's more explicit.

**Review by traderben — APPROVED** [review]

**Pull request merged**